### ISSUE TITLE
Add a link to a page explaining how to install packages

### ISSUE THREAD

**Issue by BrunoVernay** [fedora-infra/fedora-packages#279]
Pages in Fedora Wiki links to Packages on this site. (See [Software packages](https://fedoraproject.org/wiki/Help:Wiki_syntax_and_markup#Marking_Technical_Terms) )

Once here there should be a hint about what to do next: how to install the package?

There could be a simple link to an existing page:  https://fedoraproject.org/wiki/Package_management_system 
(I could improve this wiki page)

**Comment by umarbrowser** [comment]
can i take this

**Comment by umarbrowser** [comment]
Hello there i just fix the #279 issue on https://fedoraproject.org/wiki/Package_management_system

**Comment by BrunoVernay** [comment]
I don't see anything new on the packages pages (for example on https://apps.fedoraproject.org/packages/firefox  I can see no hint about how to install the package).

I guess the way to "take it" is to make a "pull request" in GitHub to propose your fix and the project owners (not me) will accept or comment it. I can reference this issue in your pull request.

**Comment by cverna** [comment]
@umarbrowser feel free to work of this issue and submit a PR. I ll be more than happy to review it.

Thanks

**Comment by steaksauce-** [comment]
What about having a section in each page page that says "Install this package" and has a code block below it saying something like
"dnf install {{ package_name }}"

**Comment by cverna** [comment]
> What about having a section in each page page that says "Install this package" and has a code block below it saying something like
> "dnf install {{ package_name }}"

Sounds like a good idea

**Comment by BrunoVernay** [comment]
Better than nothing, but I would rather point to some other (existing hopefully) page that would explain how to install with a GUI, command line, or other new methods (Flatpack, Containers, modules ...)   
Things change.

**Comment by gridhead** [comment]
Does this still need work? I would like to take this up.

(@pypingou Are you the same @pingou who wrote and maintains Pagure 😄 ?)

**Comment by pypingou** [comment]
@t0xic0der this application is basically being retired. There are a couple of options for its replacement: https://lists.fedoraproject.org/archives/list/infrastructure@lists.fedoraproject.org/thread/JLKMBMMGR6H6OSLW35ZXODPNAQP4OIEF/ but there doesn't seem to be a decision about it yet.

Anyway, I would not recommend you to start working on this feature in this code base, not worth it for you to spend time in it ;-)

> (@pypingou Are you the same @Pingou who wrote and maintains Pagure smile ?)

yes ``pingou`` was already taken in github when I registered so I went with pypingou :)

**Comment by gridhead** [comment]
Thanks, @pypingou. The easyfix site led me here so there would be a lot more rolling in with questions if the issue isn't shut. :)

**Comment by pypingou** [comment]
Good point, I've removed the project from this page (in addition to a few others that are also basically being retired): https://fedoraproject.org/w/index.php?title=Easyfix&type=revision&diff=588548&oldid=574746
